### DISCUSSION TITLE
Possibilité de se connecter en qualité d'Intervenant ou référent structure

### DISCUSSION THREAD

**Discussion by SandrineRobert** [unistra/esup-immersup-demandes#15] [7- Autres sujets]
Serait-il possible de contourner la connexion automatique LDAP pour pouvoir se connecter en qualité d'Intervenant ou de Référent Structure (=profils de son propre établissement) ?

**Comment by efinkbeiner5** [comment]
Pouvez vous nous dire dans quel cas les personnes ne seraient pas dans votre LDAP ? Chez nous, tout utilisateur qui doit se connecter à une application est forcément dans le SI.

**Comment by efinkbeiner5** [comment]
En attente de détails

**Reply by efinkbeiner5** [reply]
C'est en fait prendre l'identité de la personne.

**Comment by SandrineRobert** [comment]
Bonjour Emilie 

C'est la même chose pour l'université de Bordeaux. Je ne vois donc pas de cas particulier où les personnes ne seraient pas dans le LDAP. 
Bien cordialement 


[ https://www.u-bordeaux.fr/ ] [ mailto: ] 

[ mailto: | Sandrine ROBERT ] 
Coordinatrice de projets et appui pédagogique 
MAOIP (Mission d'Appui à l'Orientation et à l'Insertion Professionnelle - Pôle FIPVU) 

166 cours de l'Argonne 
33000 Bordeaux 
T. 05 57 57 30 54 



De: "efinkbeiner5" ***@***.***> 
À: "unistra/esup-immersup-demandes" ***@***.***> 
Cc: "Sandrine Robert" ***@***.***>, "Author" ***@***.***> 
Envoyé: Vendredi 15 Décembre 2023 11:20:42 
Objet: Re: [unistra/esup-immersup-demandes] Possibilité de se connecter en qualité d'Intervenant ou référent structure (Discussion #15) 





Pouvez vous nous dire dans quel cas les personnes ne seraient pas dans votre LDAP ? Chez nous, tout utilisateur qui doit se connecter à une application est forcément dans le SI.

**Comment by efinkbeiner5** [comment]
Déployé dans la version 4 usurpation d'identité.
